src/llm_vm/agents/REBEL/agent.py

```diff
@@ -186,7 +186,7 @@ def __init__(self, openai_key, tools, bot_str="", verbose=4):
             self.bot_str = bot_str
         else:
             self.bot_str = "<GLOBAL>" + bot_str + "<GLOBAL>"
-
+        self.nlp=spacy.load("en_core_web_md")
         # set all the API resource keys to make calls
```

Comment:
wasn't there already a load thats done globally you can use?
is this going to run on first use or at object construction time?